test/metrics/metrics_test.go

```diff
@@ -259,7 +259,7 @@ func TestVerificationRequiredMetric(t *testing.T) {
 	memberAwait := awaitilities.Member1()
 	memberAwait2 := awaitilities.Member2()
 	route := hostAwait.RegistrationServiceURL
-	hostAwait.UpdateToolchainConfig(t, testconfig.AutomaticApproval().Enabled(false)) // disable automatic approval so that users are created with verification required
+	hostAwait.UpdateToolchainConfig(t, testconfig.AutomaticApproval().Enabled(true))
```

Comment:
Since we moved to label, thats the reason, checking with automatic approval would work?

Reply:
I'm sorry, but I don't follow. What exactly are you asking about? Why changing the approval strategy, or something else?

Reply:
Yes why we switched from automatic approval false to true

Reply:
The reason is that we don't track the provision time of manually approved UserSignups. If it stayed `false` then provision time wouldn't be tracked because of this, but this is already verified in different test case. And we would need to also approve the user manually when the phone number is verified. 
Changing it to `true` let me test that the provision time won't be tracked for this user only because it went through the phone verification step.

Reply:
Thanks for the explanation.